docs/dev/storage-engine.md

```diff
@@ -0,0 +1,16 @@
+## Storage Engine
+The storage engine component is responsible for handling data points while converting them from their initial uncompressed 
```

Comment:
Replace `handling` with `manage` for consistency.

Reply:
Fixed.